Makefile

```diff
@@ -111,30 +113,54 @@ docker-test-run: # Test local docker container with test fixture notebook
 # Terraform
 ####################################
 
-### Terraform-generated Developer Deploy Commands for Dev environment           ###
-dist-dev: ## Build docker container (intended for developer-based manual build)
-	docker build --platform linux/amd64 \
-	    -t $(ECR_URL_DEV):latest \
-		-t $(ECR_URL_DEV):`git describe --always` \
-		-t $(ECR_NAME_DEV):latest .
+### Terraform-generated Developer Deploy Commands for Dev environment ###
+check-arch:
+	@ARCH_FILE=".aws-architecture"; \
+	if [[ "$(CPU_ARCH)" != "linux/amd64" && "$(CPU_ARCH)" != "linux/arm64" ]]; then \
+        echo "Invalid CPU_ARCH: $(CPU_ARCH)"; exit 1; \
+    fi; \
+	if [[ -f $$ARCH_FILE ]]; then \
+		echo "latest-$(shell echo $(CPU_ARCH) | cut -d'/' -f2)" > .arch_tag; \
+	else \
+		echo "latest" > .arch_tag; \
+	fi
+
+dist-dev: check-arch ## Build docker container (intended for developer-based manual build)
+	@ARCH_TAG=$$(cat .arch_tag); \
+	docker buildx inspect $(ECR_NAME_DEV) >/dev/null 2>&1 || docker buildx create --name $(ECR_NAME_DEV) --use; \
```

Comment:
@cabutlermit - so you did end up going with `buildx`?  Am I correct in remembering you were kind of waffling between approaches for a bit?  Sounds like maybe this approach won out?